src/test/java/SubstringExtractorTest.java

```diff
@@ -0,0 +1,32 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class SubstringExtractorTest {
+
+    @Test
+    public void test0() {
+        assertEquals("", SubstringExtractor.extractH1Content("<body><p>Hello World</p></body>"));
+    }
+
+    @Test
+    public void test1() {
+        assertEquals("", SubstringExtractor.extractH1Content("<body><h1>Greetings!</body>"));
+    }
+
+    @Test
+    public void test2() {
+        assertEquals("Hello World", SubstringExtractor.extractH1Content("<body><h1>Hello World</h1></body>"));
+    }
+
+    @Test
+    public void test3() {
+        assertEquals("Howdy", SubstringExtractor.extractH1Content("<body></h1><h1>Howdy</h1></body>"));
```

Comment:
Would it make more sense for ```SubstringExtractor.extractH1Content``` to throw an exception when given invalid HTML rather than finding "Howdy"?

Reply:
Yes, but then the lab task won't demonstrate using indexOf to find a string beginning at a specific position, i.e. after the <h1>. 

I'll update the description of the task though to explain that a real implementation would throw an exception, but that this example is demonstrating the various implementations of indexOf.